src/main/java/codesquad/service/AnswerService.java

```diff
@@ -0,0 +1,56 @@
+package codesquad.service;
+
+import codesquad.dao.AnswerRepository;
+import codesquad.domain.Answer;
+import codesquad.domain.User;
+import java.util.ArrayList;
+import java.util.List;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AnswerService {
+
+  private AnswerRepository answerRepository;
+
+  public AnswerService(AnswerRepository answerRepository) {
+    this.answerRepository = answerRepository;
+  }
+
+  public void addAnswer(Answer answer) {
+    answerRepository.save(answer);
+  }
+
+  public void deleteAnswer(Long id, User loginUser) {
+
+    checkWriterisUserOfSession(getAnswerById(id), loginUser);
+
+    answerRepository.deleteById(id);
+  }
+
+  public void updateAnswer(Long id, User loginUser, Answer updatedAnswer) {
+    Answer answer = answerRepository.findById(id).get();
+
+    checkWriterisUserOfSession(answer, loginUser);
+
+    answer.update(updatedAnswer);
+    answerRepository.save(answer);
+  }
+
+  public void checkWriterisUserOfSession(Answer answer, User user) {
+    if (!answer.isWriter(user)) {
+      throw new IllegalStateException("다른 사용자 입니다");
+    }
+  }
+
+  public List<Answer> getAllAnswers() {
+    List<Answer> answers = new ArrayList<>();
+    answerRepository.findAll().forEach(answers::add);
+
+    return answers;
```

Comment:
```suggestion
    return answerRepository.findAll();
```
로 구현해도 될 것 같은데 이렇게 구현하신 이유가 있으신가요?

Reply:
처음에 mustache에서 여러 객체를 한꺼번에 보여줄 때 List를 사용했습니다. 그래서 mustache에서 List만 인식하는 줄 알았네요